CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ include(cmake/Config.cmake)
 
 # we use the paths from the cmake GNUInstallDirs module as defaults
 # you can override these if you like
-# https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html
+# https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html
```

Comment:
would it not be better to link the minimum supported version?

Reply:
That seems too hard to maintain since it's so easy to forget to update random links in comments when the minimum CMake version changes.

Reply:
I don't think it's worth tracking some exact or minimum version here. Not like someone couldn't figure out what version is available to them if they need to know it.